README.md

```diff
@@ -798,6 +798,7 @@ There are also many **[editor plugins](#are-there-text-editor-plugins)**, a list
 **[npm packages that use `standard`](https://github.com/standard/standard-packages)**,
 and an awesome list of
 **[packages in the `standard` ecosystem](https://github.com/standard/awesome-standard)**.
+- **[standardx](https://github.com/standard/standardx)** - standard, with custom tweaks
```

Comment:
Can you add this in conjunction with the list? Basically move this line 5 lines up, so it's right after `semistandard`.